snippets/matter-debug/matter_debug.conf

```diff
@@ -15,8 +15,6 @@ CONFIG_LOG=y
 CONFIG_LOG_MODE_DEFERRED=y
 CONFIG_LOG_BUFFER_SIZE=40960
 CONFIG_LOG_SPEED=y
-CONFIG_LOG_BLOCK_IN_THREAD=y
-CONFIG_LOG_BLOCK_IN_THREAD_TIMEOUT_MS=50
```

Comment:
I guess a compromise would be to set `CONFIG_LOG_BLOCK_IN_THREAD_TIMEOUT_MS` to 1 but yeah most of the time it's better not to wait at all.